tests/test_tags.py

```diff
@@ -216,3 +216,15 @@ def test_crispy_addon():
         crispy_addon()
     with pytest.raises(TypeError):
         crispy_addon(bound_field)
+
+
+def test_crispy_missing_variable():
+    template = Template("{% load crispy_forms_tags %}{% crispy form %}")
+    html = template.render(Context({}))
+    assert html == ""
+
+
+def test_crispy_missing_variable_deep():
```

Comment:
I wanted to have a second test with nested variables where the first resolves correctly but the second doesn't.